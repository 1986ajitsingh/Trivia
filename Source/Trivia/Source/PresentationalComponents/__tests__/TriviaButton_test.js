import renderer from 'react-test-renderer';
import React from 'react';
import ModuleUnderTest from '../TriviaButton';

describe('renders correctly', () => {
  /* eslint-disable no-undef */
  test('normal', () => {
    /* eslint-enable no-undef */
    // Arrange/ Act
    const mockFun = jest.fn();
    const tree =
      renderer.create(<ModuleUnderTest onPress={mockFun}>Some Text</ModuleUnderTest>).toJSON();

    // Assert
    expect(tree).toMatchSnapshot();
  });
});
