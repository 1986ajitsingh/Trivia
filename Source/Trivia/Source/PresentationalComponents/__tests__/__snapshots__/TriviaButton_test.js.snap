// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly normal 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#4B0082",
      "padding": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#FFFFFF",
        "fontSize": 16,
        "fontWeight": "bold",
      }
    }
  >
    Some Text
  </Text>
</View>
`;
